typescript/schema.ts

```diff
@@ -8,22 +16,27 @@ viem types Hex and Address instead of simple strings for some schema properties.
 */
 
 const inputFile =
-  "https://raw.githubusercontent.com/cartesi/openapi-interfaces/fce8cc7fcf2d2fcc1940e048cd16fb8550b09779/rollup.yaml";
+  "https://raw.githubusercontent.com/cartesi/openapi-interfaces/9b9d6088d6e2f93c333d5500e868116d7e57ba91/rollup.yaml";
```

Comment:
Isn't this vesion tagged? 

Maybe we could have at least the tagged URL as a comment, for reference.